algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -225,6 +236,12 @@ case class BFInstance(hashes: BFHash, bits: BitSet, width: Int) extends BF {
       width)
   }
 
+  def checkAndAdd(item: String): (BF, ApproximateBoolean) = {
+    val bitsToActivate = BitSet(hashes(item): _*)
```

Comment:
We could also potentially not even add the new item after we figure it "contains" returns true in BFInstance, but our bit array wouldn't represent the presence of that element accurately if "contains" was only due to a collision.

This is not a problem if we just use this implementation, but it can be a problem if someone exports the bit array, or decides to fold or expand the Bloom filter (change capacity). We probably should explicitly add the element when we have more than one element there already.